unittest/c-library/CMakeLists.txt

```diff
@@ -1,26 +1,26 @@
 
 add_executable(test_library_open test_library_open.cpp test_main.cpp)
 target_link_libraries(test_library_open PRIVATE lammps GTest::GMock)
-add_test(LibraryOpen test_library_open)
+add_test(NAME LibraryOpen COMMAND test_library_open WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Isn't `CMAKE_CURRENT_BINARY_DIR` the default?

Reply:
https://cmake.org/cmake/help/latest/prop_test/WORKING_DIRECTORY.html#prop_test:WORKING_DIRECTORY